addons/markers/functions/fnc_getMapID.sqf

```diff
@@ -0,0 +1,17 @@
+/*
+ * Author: SzwedzikPL
+ * Returns map id
+ *
+ * Arguments:
+ * 0: Map classname <STRING>
+ *
+ * Return Value:
+ * Map ID <NUMBER>
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_map"];
+
+getNumber (configFile >> "CfgWeapons" >> _map >> QGVAR(mapID))
```

Comment:
Maybe we could use `_map` (string with the class name) as the map id, instead of requiring to constantly read from config what integer corresponds to that.